src/backend/executor/nodeDML.c

```diff
@@ -116,6 +116,14 @@ ExecDML(DMLState *node)
 		 * es_result_relations will contain the only relation.
 		 */
 		node->ps.state->es_result_relation_info = relInfo;
+
+		/*
+		 * Perform partition selection for in place update
+		 */
+		if (isUpdate && !isSplit)
+		{
+			node->ps.state->es_result_relation_info = slot_get_partition(node->cleanedUpSlot,node->ps.state);
```

Comment:
1) line is wider than 80 chars
2) no space before comma
4) braces are redundant

Reply:
These comments are still not corrected!